app/src/main/java/ahmaabdo/readify/rss/utils/NetworkUtils.java

```diff
@@ -125,35 +126,20 @@ public static void downloadImage(long entryId, String imgUrl) throws IOException
         }
     }
 
-    public static synchronized void deleteEntriesImagesCache(long keepDateBorderTime) {
+    public static synchronized void deleteEntryImagesCache(final int entryId, long keepDateBorderTime) {
```

Comment:
This method could be simplified a lot, because it is only called once and now only for entries that aren't favorites anyway.